docs/changed_files-v1-5-7.html

```diff
@@ -68,13 +68,139 @@ <h1>Zen Cart&reg; Version 1.5.7</h1>
                 Consult the online docs for Release-Specific considerations to note regarding this version. <a href="https://docs.zen-cart.com/user/upgrading/">https://docs.zen-cart.com/user/upgrading/</a><br>
                 <br>
                 A detailed list of all edits to all files can be seen on Github at:<br>
-                <a href="https://github.com/zencart/zencart/compare/v156...v1.5.7">v1.5.6...v1.5.7</a><br>
-                <a href="https://github.com/zencart/zencart/compare/v1.5.6c...v1.5.7">v1.5.6c...v1.5.7</a><br>
+                <a href="https://github.com/zencart/zencart/compare/v156...v1.5.7">v1.5.6...v1.5.7a</a><br>
+                <a href="https://github.com/zencart/zencart/compare/v1.5.6c...v1.5.7a">v1.5.6c...v1.5.7a</a><br>
+                <a href="https://github.com/zencart/zencart/compare/v1.5.7...v1.5.7a">v1.5.7...v1.5.7a</a><br>
             </p>
         </div>
     </section> <!-- End introCenter //-->
     
     <section id="changedFiles">
+        <div>
+            <h1>CHANGED Files in v1.5.7a (since v1.5.7)</h1>
+            <ul>
+               <li>ADDED: includes/modules/responsive_classic/ezpages_bar_header.php</li>
+               <li>ipn_main_handler.php</li>
+               <li>zc_plugins/DisplayLogs/v3.0.0/Installer/ScriptedInstaller.php</li>
+               <li>includes/classes/shopping_cart.php</li>
+               <li>includes/classes/vendors/PHPMailer/language/phpmailer.lang-tl.php</li>
+               <li>includes/classes/vendors/PHPMailer/language/phpmailer.lang-de.php</li>
+               <li>includes/classes/vendors/PHPMailer/language/phpmailer.lang-id.php</li>
+               <li>includes/classes/vendors/PHPMailer/VERSION</li>
+               <li>includes/classes/vendors/PHPMailer/composer.json</li>
+               <li>includes/classes/vendors/PHPMailer/get_oauth_token.php</li>
+               <li>includes/classes/vendors/PHPMailer/src/PHPMailer.php</li>
+               <li>includes/classes/vendors/PHPMailer/src/OAuth.php</li>
+               <li>includes/classes/vendors/PHPMailer/src/SMTP.php</li>
+               <li>includes/classes/vendors/PHPMailer/src/Exception.php</li>
+               <li>includes/classes/vendors/PHPMailer/src/POP3.php</li>
```

Comment:
For the `vendors/PHPMailer` directory, let's not list the files, but rather just say "entire directory". This is the typical pattern followed for code that comes from other sources.